tests/test_app/TestApp/Error/TestAppsExceptionRenderer.php

```diff
@@ -17,7 +17,7 @@ class TestAppsExceptionRenderer extends ExceptionRenderer
      */
     protected function _getController()
     {
-        if (!$request = Router::getRequest(true)) {
+        if (!$request = $this->request ?: Router::getRequest(true)) {
```

Comment:
We generally avoid assignment inside conditionals as it makes code more complex and harder to read as `=` and `==` are easily misread.

Reply:
Yes, fixed. I keep it close to the way it was originally written.